webpage/index.html

```diff
@@ -94,95 +102,88 @@ <h1>Farben</h1>
                     <form name="prog">
-                    <label for="mode-wordclock" class="pure-radio">
-                        <input id="mode-wordclock" type="radio" name="mode" value="color" checked> Wortuhr
-                    </label>
-                    <label for="mode-color" class="pure-radio">
-                        <input id="mode-color" type="radio" name="mode" value="color"> Farbe
-                    </label>
-                    <label for="mode-seconds" class="pure-radio">
-                        <input id="mode-seconds" type="radio" name="mode" value="seconds"> Sekunden
-                    </label>
-                    <label for="mode-marquee" class="pure-radio">
-                        <input id="mode-marquee" type="radio" name="mode" value="marquee"> Laufschrift
-                    </label>
-                    <label for="mode-rainbow" class="pure-radio">
-                        <input id="mode-rainbow" type="radio" name="mode" value="rainbow"> Regenbogen
-                    </label>
-                    <label for="mode-change" class="pure-radio">
-                        <input id="mode-change" type="radio" name="mode" value="change"> Farbwechsel
-                    </label>
+                        <input id="mode-wordclock" type="radio" name="mode" value="color" checked><label for="mode-wordclock" class="pure-radio" data-i18next="functions.mode.wordclock"></label>
+
+                        <input id="mode-color" type="radio" name="mode" value="color"><label for="mode-color" class="pure-radio" data-i18next="functions.mode.color"></label>
+
+                        <input id="mode-seconds" type="radio" name="mode" value="seconds"><label for="mode-seconds" class="pure-radio" data-i18next="functions.mode.seconds"></label>
+
+                        <input id="mode-marquee" type="radio" name="mode" value="marquee"><label for="mode-marquee" class="pure-radio" data-i18next="functions.mode.marquee"></label>
+
+                        <input id="mode-rainbow" type="radio" name="mode" value="rainbow"><label for="mode-rainbow" class="pure-radio" data-i18next="functions.mode.rainbow"></label>
+
+                        <input id="mode-change" type="radio" name="mode" value="change"><label for="mode-change" class="pure-radio" data-i18next="functions.mode.color-change"></label>
                     </form>
                 </div>
 
                 <div class="box" id="animation-box">
-                    <h2>Animation</h2>
+                    <h2 data-i18next="functions.animation.h2"></h2>
                     <form class="pure-form pure-form-aligned">
                         <fieldset>
                             <div class="pure-control-group">
-                                <label for="animation-types">Art</label>
+                                <label for="animation-types" data-i18next="functions.animation.type.label"></label>
                                 <select id="animation-types" size="1">
```

Comment:
The options for this select are supplied by the backend (in `clockWork.hpp`). I would like to include them in the frontend some day and remove them from the backend, so we can translate them properly. My C++/C skills are not sufficient for that task, so the select will be in German for now until we move the text to the frontend.